tests/typecheck/models/test_create.yml

```diff
@@ -53,7 +53,7 @@
                     id2 = models.AutoField(primary_key=True)
                     name2 = models.CharField(max_length=50)
 
-                class Child1(Parent1, Parent2):
+                class Child1(Parent1, Parent2):  # type: ignore[misc]
```

Comment:
Now, I've got no idea how or if types can even be solved properly for multiple inheritance.

Given that e.g. `Child.child1` exists here, and I'm not sure what that should or can even be typed as.

Anyways, this fails on the descriptor types that the plugin adds:

```python
Parent1.child1: ReverseOneToOneDescriptor[Child1, Parent1]
Parent2.child1: ReverseOneToOneDescriptor[Child1, Parent2]
```

And since it's also a python inheritance, mypy end up seeing an override on `Child1.child1` that is not compatible.

While I'm writing this I realise it's probably better to capture the error in `out:` to make everything a bit more obvious